web/app/js/components/TopModule.jsx

```diff
@@ -26,8 +27,8 @@ class TopModule extends React.Component {
       resource: PropTypes.string
     }),
     startTap: PropTypes.bool.isRequired,
-    updateNeighbors: PropTypes.func,
-    updateTapClosingState: PropTypes.func
+    updateTapClosingState: PropTypes.func,
+    updateUnmeshedSources: PropTypes.func
```

Comment:
since we now depend on this to show the tap icon in the TopTable, I think we should make this `isRequired`

Reply:
When building  `<TopModule>` in the `/top` page I'm not passing any function because there's no octopus that needs an `unmeshedSources` object to be available.

Reply:
ah, right! I thought we needed this to keep track of the unmeshed upstreams for the table, but we're keeping track of them in TopModule